Integrations/Integration-Wildfire.yml

```diff
@@ -0,0 +1,222 @@
+    var headers = {'Content-Type': ['application/x-www-form-urlencoded']};
+
+    switch (command) {
+      case 'test-module':
+          url += urlDict.report;
+          var res = http(
+              url,
+              {
+                  Method: 'POST',
+                  Body: 'apikey='+params.token+'&format=xml&hash=7f638f13d0797ef9b1a393808dc93b94',
+                  Headers: headers
+              },
+              params.insecure,
+              params.proxy
+          );
+
+          if(res.StatusCode===200)
+              return 'ok';
+          else
+              return 'Something went wrong.\n Status code: ' + res.StatusCode + '\nBody: '+ JSON.stringify(res) + '.';
+          break;
+
+      case 'WildFireReport':
+        url += urlDict.report;
+        var hash = args.md5 || args.sha256;
+        if(!hash)
+            throw 'Missing md5/sha256';
+
+        var bodyXML = 'apikey='+params.token+'&format=xml&hash='+hash;
+        var resXML = sendRequest(url, bodyXML, headers);
+        var result = JSON.parse(x2j(resXML));
+        var report = result.wildfire.task_info.report;
+        var file_info = result.wildfire.file_info;
+
+        var context = parseFile(result.wildfire.file_info);
+
+        if(args.format==='pdf'){
+          var bodyPDF = 'apikey='+params.token+'&format=pdf&hash='+hash;
+          var resPDF = sendRequest(url, bodyPDF, headers);
+
+          var currentTime = new Date();
+          var fileName = command + '_at_' + currentTime.getTime();
+
+          return {Type: entryTypes.note,
+                  FileID: saveFile(resPDF),
+                  File: fileName,
+                  Contents: fileName,
+                  EntryContext: context
+          };
+        }
+        else{ /* default format is xml*/
+          var md = resToMD(file_info,'File info');
+          for (var i=0; i<report.length; i++){
+              md += resToMD(report[i],'Report '+i);
+          }
+          return {Type: entryTypes.note,
+                  Contents: result,
+                  ContentsFormat: formats.json,
+                  HumanReadable: md,
+                  ReadableContentsFormat: formats.markdown,
+                  EntryContext: context
+          };
+        }
+        break;
+
+      case 'WildFireUpload':
+          url += urlDict.upload;
+          var result = sendMultipartRequest(url, headers, args.upload, params.token);
+          var context = parseFile(result.wildfire['upload-file-info']);
+          var md = resToMD(result.wildfire['upload-file-info'],'Wildfire Report');
+
+          return {Type: entryTypes.note,
+                      Contents: result,
+                      ContentsFormat: formats.json,
+                      HumanReadable: md, /*todo*/
+                      ReadableContentsFormat: formats.markdown,
+                      EntryContext: context
+              };
+
+    }
+
+  type: javascript
+  commands:
+  - name: WildFireReport
```

Comment:
You can use outputs instead of creating an entry context and returning it, but in order to be used in playbooks you should add the output to context path File